### ISSUE TITLE
/stats

### ISSUE THREAD

**Issue by FekkOffItsHamppe** [WSS-tracking/WSS#99]
Would be nice with /stats

**Comment by dev-bas** [comment]
Could you elaborate on what you think should happen when someone does /stats?

**Comment by FekkOffItsHamppe** [comment]
Show how many Kills/arrest/deaths you have. Hr done, br done and so on

**Comment by Liyones** [comment]
i think this idea is done in other server already , why don't u make something different when clicking on f5 it will open a ^panel with all ur infos  arrests/deaths/kills/job/groups u added in/punishments number.... etc

**Comment by WSSLidl** [comment]
I don't think we currently save kills, arrests etc. So this requires some more work than just making a command. I like the idea tho
